### PR TITLE
Create a global modal

### LINKED ISSUE #3030: Pop-up
# Brief
We need a full page pop-up to let users know about an awesome new thing

# Content
- Video/Gif (TBD)
- Headline (TBD)
- Body text (TBD)
- CTA (TBD)

# Special considerations
- The video/gif will need to be swapped out quite last minute. Maybe we use an outside link with dummy content and change it when we have final content. This way we don't need to submit a new release
- This will need to go live on the 21st or 22nd the latest. This probably means that we need to adjust our release schedule a little bit?

# Figma file
https://www.figma.com/file/q8gfu9XTbYBF07KMnDCtsD/UI---Dark?node-id=16973%3A314509&t=z4WSTGlWoehlteK2-4

### PR DESCRIPTION
**Pull request by kkosiorowska**
Closes #3030 

## UI
Screenshot 2023-02-17 at 13 47 47

## To Test
- [ ] Check if the global modal is visible.

Latest build: [extension-builds-3038](https://github.com/tallyhowallet/extension/suites/11048787545/artifacts/561427305) (as of Fri, 17 Feb 2023 14:57:32 GMT).

### CODE REVIEW

**Comment by 0xDaedalus** [comment]
Updated with new Copy: 
image

File: background/redux-slices/migrations/to-26.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+export default (prevState: Record<string, unknown>): NewState => {
+  const typedPrevState = prevState as OldState
+
+  return {
+    ...prevState,
+    ui: {
+      ...typedPrevState.ui,
+      showGlobalModal: true,
```
Comment (jagodarybacka) [inline background/redux-slices/migrations/to-26.ts:23]: Not a blocker but to be improved later - tbh I'm not sure if doing it based on the redux state is the best way, every time we will want to show this modal to users on the new release automatically we will have to do the migration right?

I think making it based on the local storage (like we keep the notification banners closed by saving their id in the local storage) for example would make it easier to reuse

Comment (kkosiorowska) [reply background/redux-slices/migrations/to-26.ts:23]: Great suggestion!

File: ui/pages/Popup.tsx [hunk 2]
```diff
@@ -159,6 +160,7 @@ export function Main(): ReactElement {
       <div className="top_menu_wrap_decoy">
         <TopMenu />
       </div>
+      <GlobalModal id="meet_taho" />
```
Comment (jagodarybacka) [inline ui/pages/Popup.tsx:163]: Wohoo 🎉 it is so much simpler now!

**Review by jagodarybacka — APPROVED** [review]
LGTM ✨

**Pull request merged**